recipes-extended/jailhouse/jailhouse_0.10.bb

```diff
@@ -55,23 +62,28 @@ USER_SPACE_CFLAGS = '${CFLAGS} -DLIBEXECDIR=\\\"${libexecdir}\\\" \
 TOOLS_SRC_DIR = "${S}/tools"
 TOOLS_OBJ_DIR = "${S}/tools"
 
+B = "${S}"
+
```

Comment:
Why? Couldn't this to be fixed?

Reply:
> Why? Couldn't this to be fixed?

I'll check again, at a first look wasn't

Reply:
I'd rather keep this, since build is looking for setup.py which is present in {S}